modules/cram/templates/concat_vcf.sh

```diff
@@ -7,21 +7,41 @@ concat () {
   args+=("--allow-overlaps")
   args+=("--remove-duplicates")
   args+=("--output-type" "z")
-  args+=("--output" "!{vcfOut}")
+  args+=("--output" "unsorted_!{vcfOut}")
   args+=("--no-version")
   args+=("--threads" "!{task.cpus}")
-  args+=(!{vcfs})
+  for vcf in !{vcfs}
+  do
+    args+=("sorted_${vcf}")
+  done
 
   ${CMD_BCFTOOLS} "${args[@]}"
 }
 
+bcftools_sort () {
+  ${CMD_BCFTOOLS} sort --output-type z "unsorted_!{vcfOut}" --output "!{vcfOut}"
```

Comment:
add `--no-version`
add `--threads "!{task.cpus}"`?

Reply:
sort doesn't have a `--no-version` according to the error message

Reply:
and no --threads